schemes/rrtmgp/rrtmgp_pre_namelist.xml

```diff
@@ -0,0 +1,329 @@
+    </desc>
+    <values>
+      <value>.true.</value>
+    </values>
+  </entry>
+  <entry id="is_offline_dyn">
+    <type>logical</type>
+    <category>rrtmgp</category>
+    <group>rrtmgp</group>
+    <standard_name>is_offline_dynamical_core</standard_name>
+    <units>flag</units>
+    <desc>
+      If true, we are using the offline dynamical core / specified dynamics
+    </desc>
+    <values>
+      <value>.false.</value>
+    </values>
+  </entry>
+  <entry id="spectralflux">
+    <type>logical</type>
+    <category>rrtmgp</category>
+    <group>rrtmgp</group>
+    <standard_name>do_up_down_flux_per_band_diagnostic</standard_name>
+    <units>flag</units>
+    <desc>
+       If true, calculate the up and down fluxes per band; used by CARMA
+    </desc>
+    <values>
+      <value>.false.</value>
+    </values>
+  </entry>
+  <entry id="graupel_in_rad">
+    <type>logical</type>
+    <category>rrtmgp</category>
+    <group>rrtmgp</group>
+    <standard_name>do_calculate_radiative_effect_of_graupel</standard_name>
+    <units>flag</units>
+    <desc>
+       Flag to use graupel cloud fraction in radiation
+    </desc>
+    <values>
+      <value>.false.</value>
+    </values>
+  </entry>
+  <entry id="nswbands">
+    <type>integer</type>
+    <category>rrtmgp</category>
+    <group>rrtmgp</group>
+    <standard_name>number_of_bands_for_shortwave_radiation</standard_name>
+    <units>count</units>
+    <desc>
+       Number of shortwave bands
+    </desc>
+    <values>
+      <value>14</value>
+    </values>
+  </entry>
+  <entry id="nlwbands">
+    <type>integer</type>
+    <category>rrtmgp</category>
+    <group>rrtmgp</group>
+    <standard_name>number_of_bands_for_longwave_radiation</standard_name>
+    <units>count</units>
+    <desc>
+       Number of longwave bands
+    </desc>
+    <values>
+      <value>16</value>
+    </values>
+  </entry>
+  <entry id="nradgas">
+    <type>integer</type>
+    <category>rrtmgp</category>
+    <group>rrtmgp</group>
+    <standard_name>number_of_active_gases_for_RRTMGP</standard_name>
+    <units>count</units>
+    <desc>
+       Number of radiatively active gases for RRTMGP
+    </desc>
+    <values>
+      <value>8</value>
+    </values>
+  </entry>
+  <entry id="gasnamelength">
+    <type>integer</type>
+    <category>rrtmgp</category>
+    <group>rrtmgp</group>
+    <standard_name>character_length_of_list_of_active_gases_for_RRTMGP</standard_name>
+    <units>count</units>
+    <desc>
+       Character length for each element of the list of radiatively active gases.
+    </desc>
+    <values>
+      <value>5</value>
+    </values>
+  </entry>
+  <entry id="p_top_for_rrtmgp">
+    <type>real</type>
+    <kind>kind_phys</kind>
+    <category>rrtmgp</category>
+    <group>rrtmgp</group>
+    <standard_name>air_pressure_cutoff_for_rrtmgp</standard_name>
+    <units>Pa</units>
+    <desc>
+      Top pressure level for RRTMGP
+         Default: 1.0 for WACCM/WACCMX
+         10. for all other CAM runs
```

Comment:
Should this actually be two Pa?

```suggestion
         2.0 for all other CAM runs
```

Reply:
checked vs. CAM and it's 5 for non-WACCM runs with RRTMGP now. Changed to that!